src/service/controllers/auth.js

```diff
@@ -180,10 +180,10 @@ exports.signin = (req, res) => {
 }
 
 exports.start = (req, res) => {
-	var redirectUrl = req.query.redirect_uri;
-	redirectUrl += '?code=' + req.query.code
+	var redirectUrl = new URL(req.query.redirect_uri);
```

Comment:
The method does support query parameters in the url now.
Before, it was adding **?code**, instead of **&code** if the UI had query parameters .